packages/components/src/buttons/src/Button.module.css

```diff
@@ -103,11 +143,24 @@
     background-color: var(--background-color);
     color: var(--color);
     border: var(--border);
-    align-items:center
+    align-items:center;
+    text-decoration: none;
+}
+
+.hop-Button--icon-only {
+    column-gap: 0;
```

Comment:
Why specifying column-gap only, there's now row gap, as far as I know

Reply:
what do you mean? i set column-gap in Button, and i unset it in -icon-only